data/stig/baseline_comparison.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/python3
+
+# Extract baseline and save baseline in a hash map
+def extractBaseline(file):
+    stdout = open(file, "r+")
+    line = stdout.readline()
+    tmp = {}
```

Comment:
It is good to use parameter ret instead of tmp, I think.

Reply:
variable name changed